cpp/client/ct.cc

```diff
@@ -202,14 +202,14 @@ static bool PrecertChainToEntry(const cert_trans::PreCertChain& chain,
     return false;
   }
 
-  Cert::Status status =
+  StatusOr<bool> status =
```

Comment:
Couple of while-you're-here ones:

Constable, constable?
/status/has_poison/ ?